Library/Formula/teensy_loader_cli.rb

```diff
@@ -1,9 +1,10 @@
 class TeensyLoaderCli < Formula
   desc "Command-line integration for Teensy USB development boards"
   homepage "https://www.pjrc.com/teensy/loader_cli.html"
+  depends_on "libusb-compat"
```

Comment:
Oh, not in the head block, just below it and above the install block.  Otherwise `brew audit --strict teensy_loader_cli` will complain.